Assets/MixedRealityToolkit/_Core/Devices/VoiceInput/WindowsDictationInputDeviceManager.cs

```diff
@@ -116,14 +120,17 @@ public override async void Disable()
                 dictationRecognizer.DictationComplete -= DictationRecognizer_DictationComplete;
                 dictationRecognizer.DictationError -= DictationRecognizer_DictationError;
             }
+#endif
         }
 
         public override void Destroy()
         {
+#if UNITY_STANDALONE_WIN || UNITY_WSA || UNITY_EDITOR_WIN
```

Comment:
Same thing here. Now that we wrapped the whole class we don't need this in every method.